src/static/bitbucket/components/App.tsx

```diff
@@ -236,6 +245,8 @@ const App = () => {
         onCheckAgainClicked={onCheckAgainClicked}
         onLandWhenAbleClicked={onLandWhenAbleClicked}
         onLandClicked={onLandClicked}
+        isSquashMergeChecked={isSquashMergeChecked}
+        onChange={onChange}
```

Comment:
nit: Looks like this was was missed out? `onMergeStrategyChange` ?

Reply:
Done